source/lac/solver_control.cc

```diff
@@ -407,10 +415,16 @@ ConsecutiveControl::check (const unsigned int step,
       n_converged_iterations++;
       if (n_converged_iterations == n_consecutive_iterations)
         {
+          if (this->m_log_result)
+            deallog << "Number of consecutively converged iterations " << n_converged_iterations << std::endl;
```

Comment:
oh, please remove these lines of output to `deallog` again -- we agreed in principle in #4730 to get rid of all of this output